examples/sync/src/bin/client.rs

```diff
@@ -101,18 +101,21 @@ where
         context: context.clone(),
         db_config,
         fetch_batch_size: NonZeroU64::new(config.batch_size).unwrap(),
-        target_hash,
-        lower_bound_ops: oldest_retained_loc,
-        upper_bound_ops: latest_op_loc,
+        target: SyncTarget {
+            hash: target_hash,
```

Comment:
root might be a better name if this is indeed some root hash?  (i use root implicitly for "root_hash" elsewhere)

Reply:
Per Patrick's request, I've renamed all "root_hash" and "root_digest" to just "root" in variable and function names in #1299